### PR TITLE
Run new_gallery__transition_perf on mokey in staging

### PR DESCRIPTION
**Pull request by zanderso**

### CODE REVIEW

File: .ci.yaml [hunk 1]
```diff
@@ -2796,6 +2807,28 @@ targets:
...
 
+  # Mokey, Impeller
+  - name: Linux_mokey new_gallery_impeller__transition_perf
+    recipe: devicelab/devicelab_drone
+    bringup: true # Device exists only in staging.
+    presubmit: false
+    timeout: 60
+    properties:
+      tags: >
+        ["devicelab", "android", "linux", "mokey"]
```
Comment (yusuf-goog) [inline .ci.yaml:2818]: Looks like the tags are going to be the same for all linux_mokey builds ? Can they live in the linux_mokey section?

**Review by yusuf-goog — APPROVED** [review]
LGTM with one comment.

File: .ci.yaml (hunk 1 above)
Comment (zanderso) [reply .ci.yaml:2818]: I don't really know how the .ci.yaml file is supposed to work, and I'm just following suit with every other build. If putting the common tags under the `linux_mokey` section is expected to work, I can try that cleanup for the whole file in a separate PR.

**Comment by zanderso** [comment]
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_mokey%20new_gallery_impeller__transition_perf/1/overview
https://ci.chromium.org/ui/p/flutter/builders/staging/Linux_mokey%20new_gallery__transition_perf/1/overview

**Pull request merged**